Demos/CaptureASL/CaptureASL/ViewController.m

```diff
@@ -8,38 +8,41 @@
 #import "ViewController.h"
 #import <asl.h>
 
-@implementation ViewController
-{
+@implementation ViewController {
     int count;
     aslclient client;
 }
 
-- (void)awakeFromNib
-{
+- (void)awakeFromNib {
     [super awakeFromNib];
-    
+
     count = 0;
     client = asl_open(NULL, "CocoaLumberjack.CaptureASL", 0);
 }
 
-- (IBAction)log:(id)sender
-{
+- (IBAction)log:(id)sender {
     NSLog(@"ASL test message %d", count++);
 }
 
-- (IBAction)asl_log:(UIButton *)sender
-{
+- (IBAction)asl_log:(UIButton *)sender {
     char *level = NULL;
-    switch (sender.tag)
-    {
-        case ASL_LEVEL_EMERG   : level = "ASL_LEVEL_EMERG";   break;
-        case ASL_LEVEL_ALERT   : level = "ASL_LEVEL_ALERT";   break;
-        case ASL_LEVEL_CRIT    : level = "ASL_LEVEL_CRIT";    break;
-        case ASL_LEVEL_ERR     : level = "ASL_LEVEL_ERR";     break;
-        case ASL_LEVEL_WARNING : level = "ASL_LEVEL_WARNING"; break;
-        case ASL_LEVEL_NOTICE  : level = "ASL_LEVEL_NOTICE";  break;
-        case ASL_LEVEL_INFO    : level = "ASL_LEVEL_INFO";    break;
-        case ASL_LEVEL_DEBUG   : level = "ASL_LEVEL_DEBUG";   break;
+
+    switch (sender.tag) {
+        case ASL_LEVEL_EMERG: level = "ASL_LEVEL_EMERG";   break;
+
+        case ASL_LEVEL_ALERT: level = "ASL_LEVEL_ALERT";   break;
+
+        case ASL_LEVEL_CRIT: level = "ASL_LEVEL_CRIT";    break;
+
+        case ASL_LEVEL_ERR: level = "ASL_LEVEL_ERR";     break;
+
+        case ASL_LEVEL_WARNING: level = "ASL_LEVEL_WARNING"; break;
+
+        case ASL_LEVEL_NOTICE: level = "ASL_LEVEL_NOTICE";  break;
+
+        case ASL_LEVEL_INFO: level = "ASL_LEVEL_INFO";    break;
+
+        case ASL_LEVEL_DEBUG: level = "ASL_LEVEL_DEBUG";   break;
```

Comment:
I see no reason to introduce newlines to all the switch statements. They are already quite easy to read. This applies to all other switch statements too.

Reply:
@hhanesand 
Here I strongly suggest pure functional style.

```objective-c
- (char *)levelFromTag:(NSInteger)tag {
    switch (tag) {
        case ASL_LEVEL_EMERG   : return "ASL_LEVEL_EMERG";
        case ASL_LEVEL_ALERT   : return "ASL_LEVEL_ALERT";
        case ASL_LEVEL_CRIT    : return "ASL_LEVEL_CRIT";
        case ASL_LEVEL_ERR     : return "ASL_LEVEL_ERR";
        case ASL_LEVEL_WARNING : return "ASL_LEVEL_WARNING";
        case ASL_LEVEL_NOTICE  : return "ASL_LEVEL_NOTICE";
        case ASL_LEVEL_INFO    : return "ASL_LEVEL_INFO";
        case ASL_LEVEL_DEBUG   : return "ASL_LEVEL_DEBUG";
        default                : return NULL;
    }
}
```